ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -211,6 +211,39 @@ def test_tensorflow_slogdet(
     )
 
 
+# cholesky_solve
+@handle_cmd_line_args
+@given(
+    dtype_and_x=_solve_get_dtype_and_data(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.cholesky_solve"
+    ),
+)
+def test_tensorflow_cholesky_solve(
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    data1, data2 = dtype_and_x
```

Comment:
I see you have liberally tried to copy the test for the `solve` function. However that function itself isn't passing its tests. You might want to check out the Ivy Tests for the solve function in our ivy framework [here](https://github.com/unifyai/ivy/blob/c6e3c7a24f63fe7360a6977a3ca89b3ae24d9ad5/ivy_tests/test_ivy/test_functional/test_core/test_linalg.py#L621) to understand how to implement tests for function like these. Along with the [docs](https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends_tests.html#ivy-frontend-tests)

Reply:
@Auxia Thanks! Can you explain the difference between the functional and frontend tests?

Reply:
@Auxia Hi thanks for the link to the ivy test for solve! I'm trying to use the same approach here. As I understand it, rhs can be the same as get_second_matrix() but I need to modify get_first_matrix() to return a symmetric positive definite matrix. I'm facing some Assertion Errors with the function right now and was hoping you can help me debug it? Thanks!

Reply:
Thanks for the advice! I've updated the test to produce the cholesky decomposition of a symmetric positive definite matrix for testing. All tests are passing now!